### ISSUE TITLE
No credit

### ISSUE THREAD

**Issue by B4rabbas** [BGforgeNet/Fallout2_Restoration_Project#183]
### What happened
When I click on "credits" at main menu, screen fade out, then fade in back to the main menu
### What you expected to happen
When I click on "credits" at main menu, credits are showed
### Screenshot
![scr00000](https://user-images.githubusercontent.com/36156012/215624877-cb70acc5-dae4-4560-a698-411a179d75ba.png)
### Savegame
[SAVEGAME.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10541068/SAVEGAME.zip)

**Comment by NovaRain** [comment]
Some translation packages (French, Italian, Portuguese and Hungarian) are missing `credits.txt` in their `text\<language>\` folder.

**Comment by burner1024** [comment]
Right, drop [this file](https://github.com/BGforgeNet/Fallout2_Restoration_Project/blob/master/data/text/english/credits.txt) into `data/text/french` and see if it works then.

Menu font is so ugly though... Is how it always looked in French?

**Comment by B4rabbas** [comment]
Thanks for yours answers.
Copying credit.txt fix my issue.

@burner1024 The menu font should be identical to English version, I don't know why it looks like this.

**Comment by B4rabbas** [comment]
Here is a french translated credit.txt
[credits.txt](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10545157/credits.txt)

**Comment by B4rabbas** [comment]
For the font I just copy the folder content of this https://github.com/BGforgeNet/Fallout2_Restoration_Project/tree/master/data/text/french
No more font issue. With integrated sfall hires, some string length is too long for the interface.
So I've modified the main menu in misc.msg , "NOUVEAU JEU" & "CHARGER JEU" are just a bit to long, I've just removed "JEU"  (and some options, "TERMINER" -> "OK", "ANNULER" -> "RETOUR").
![scr00002](https://user-images.githubusercontent.com/36156012/215748181-b615600d-7c95-45b1-ae24-78e102d5a617.png)
[misc&options.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10545932/misc.options.zip)

**Comment by burner1024** [comment]
How do Options look?

**Comment by B4rabbas** [comment]
I've also modified (in options.msg) : 
"Aucune préfèrence." -> "Aucune" (or the button can't work)
"INJURES AU COMBAT" -> "INJURES COMBAT" (get out of box)
![scr00003](https://user-images.githubusercontent.com/36156012/215863606-65a53677-cf18-4475-b0bc-485a75f42f63.png)

If you wanna look the result : 
![scr00004](https://user-images.githubusercontent.com/36156012/215863789-1309688f-3e43-45ad-8227-8807f10dc282.png)

[options&misc.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10550131/options.misc.zip)

**Comment by burner1024** [comment]
It's `font4.aaf`, but upon using original one it looks different on my system.

![Captura desde 2023-02-01 02-40-26](https://user-images.githubusercontent.com/1279852/215865488-6a319cb8-eef1-4712-8bbb-26e9583c28ed.png)

**Comment by B4rabbas** [comment]
Are you using Mash's Hi-res patch or the builtin one from sfall ?

**Comment by burner1024** [comment]
Mash

**Comment by B4rabbas** [comment]
I'm not able to reproduce it, maybe it's because I've updated the ddraw.dll from latest version of sfall.

**Comment by burner1024** [comment]
Hmm, I also restored font3.aaf, and now it's more similar. Strange, not sure why they are connected.
![Captura desde 2023-02-01 02-57-51](https://user-images.githubusercontent.com/1279852/215869574-55ff1ef0-8ce9-458a-b707-8b4ae99e04af.png)

**Comment by B4rabbas** [comment]
files should be loaded by priority

**Comment by B4rabbas** [comment]
Does font2.aaf is restored, I'm asking myself if my modifications are pointless.

**Comment by burner1024** [comment]
font1 and font2 were default anyway, french translation only includes modified 3 and 4.

**Comment by B4rabbas** [comment]
Fine, thanks for your attention and for maintaining this project.

**Comment by burner1024** [comment]
Let's keep it open for now.

**Comment by B4rabbas** [comment]
I've done the same for some others menus (skilldex, lsgame and editor), I think it's all done for the UI minor issues.
[french.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10551863/french.zip)
(here is the completed french archive)

I've integrated the existing French versions to the portraits descriptions.
[premade.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10552053/premade.zip)

**Comment by burner1024** [comment]
The issue is due to broken fonts in the repo. I added proper ones, now the original translation looks better:

![Captura desde 2023-02-01 11-35-45](https://user-images.githubusercontent.com/1279852/215950224-6c91a1d0-1f37-49b6-a94e-ea905d718914.png)

I included your `credits.txt`, though.

There's also some splash screens and frms that were missing, now also included.

**Comment by B4rabbas** [comment]
Can you share build instructions ?

**Comment by burner1024** [comment]
Builds are available in [actions](https://github.com/BGforgeNet/Fallout2_Restoration_Project/actions) tab ([ex](https://github.com/BGforgeNet/Fallout2_Restoration_Project/actions/runs/4060983324)).

They are for testing, not playing.

**Comment by B4rabbas** [comment]
It look so great, thanks brother. :)

There is only one string to change in options.msg
![scr00000](https://user-images.githubusercontent.com/36156012/216054389-9072c73f-9d90-4176-9fe8-28f59abba8cf.png)

Need to change (lign 43) {214}{}{Aucune préference.} -> {214}{}{Aucune}
![scr00000](https://user-images.githubusercontent.com/36156012/216054439-edaa259d-2a42-4aa2-8cb9-e4800a978db7.png)

[options.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/10557382/options.zip)

**Comment by burner1024** [comment]
Translation is at [Hive](https://hive.bgforge.net/projects/fallout/rp/fr/), you can change lines.

**Comment by burner1024** [comment]
Don't close yet, it needs more work in other translations.
